workers/api-server/src/http/admin.rs

```diff
@@ -451,14 +457,6 @@ impl TypedHandler for AdminAssignOrderToMatchingPoolHandler {
         let order_id = parse_order_id_from_params(&params)?;
         let matching_pool = parse_matching_pool_from_url_params(&params)?;
 
-        // Check that the order exists. We do this by checking the order
```

Comment:
we remove the order-metadata-based order existence check here, and below, as these admin endpoints should not depend on historical state being enabled. there's no other clean way to check for an order's existence, as indicated by the doc comment, so IMO it is fine to simply remove the check and revisit when we implement the tracking of order metadata for actively managed orders